frontend/src/components/Sheet/Sheet.module.scss

```diff
@@ -16,8 +16,6 @@
   display: flex;
   flex-direction: row;
   justify-content: center;
-  z-index: -1;
```

Comment:
these bits were to make sure the gamemodes sheet appears behind the mobile navbar - were they causing other problems

Reply:
yeah hover didnt work at all

Reply:
reapplying these properties doesnt seem to change anything on my end unless im missing smth? maybe my screen is too big?

![image](https://github.com/user-attachments/assets/46ff8093-d020-4fd6-bd13-612ad2bcc45a)
![image](https://github.com/user-attachments/assets/d7b2ff51-4339-4df4-b6a6-ad00f0c61790)

Reply:
doesnt seem to do much on iphone xr
![image](https://github.com/user-attachments/assets/6be5cd07-3195-4543-9d2f-c7aa51f4d233)
![image](https://github.com/user-attachments/assets/f0584f4b-a0a3-4482-bee1-7145df703d5b)

Reply:
<img width="704" alt="Screenshot 2024-09-13 at 5 25 44 pm" src="https://github.com/user-attachments/assets/209f7fd8-a408-4688-83d0-2e2f5931ac0c">
<img width="753" alt="Screenshot 2024-09-13 at 5 25 49 pm" src="https://github.com/user-attachments/assets/5d98bed2-78f7-4b94-909f-2c306394a375">